payu/archive_scripts/build_intake_ds.py

```diff
@@ -0,0 +1,57 @@
+#!python3
+# Copyright 2024 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: Apache-2.0
+# modules:
+# use:
+#     - /g/data/hh5/public/modules
+# load:
+#     - conda/analysis
+
+from access_nri_intake.source import builders
+import os
+
+from pathlib import Path
+import sys
+
+path_root = Path(__file__).parents[2]
+sys.path.append(str(path_root))
+
+from scripts_common import get_provenance_metadata, md5sum
+
+from ruamel.yaml import YAML
+from ruamel.yaml.comments import CommentedMap
+
+METADATA_FILENAME = "metadata.yaml"
+UUID_FIELD = "experiment_uuid"
+
+builder = builders.AccessOm3Builder(path="archive/")
+
+builder.build()
+
+# Log invalid assets
+builder.invalid_assets
+
+# Get experiment uuid
+# follows https://github.com/payu-org/payu/blob/ef55e93fe23fcde19024479c0dc4112dcdf6603f/payu/metadata.py#L90
+metadata_filename = Path(METADATA_FILENAME)
+if metadata_filename.exists():
+    metadata = CommentedMap()
+    metadata = YAML().load(metadata_filename)
+    uuid = metadata.get(UUID_FIELD, None)
+else:
+    warn(f"{METADATA_FILENAME} not found in archive folder")
```

Comment:
`warn` is not defined

Reply:
Is the metadata worth including in the filename? I would still be in the description already.

Its easier to generalise opening $run_dir/intake_ds.json than $run_dir/intake_ds_#######.json ?

Reply:
Yeah true - let's just remove it. Could we call it `intake_esm_ds` though?